lib/widgets/layouts/adaptive_layout/adaptive_scaffold.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:async/async.dart';
+import 'package:fluffychat/pages/chat_list/client_chooser_button.dart';
+import 'package:fluffychat/widgets/layouts/adaptive_layout/adaptive_scaffold_view.dart';
+import 'package:fluffychat/widgets/layouts/enum/adaptive_destinations_enum.dart';
+import 'package:fluffychat/widgets/matrix.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:matrix/matrix.dart';
+
+typedef OnTapSearch = Function();
+typedef OnCallbackFromSearch = Function();
+typedef OnClientSelectedSetting = Function(Object object, BuildContext context);
+typedef OnDestinationSelected = Function(int index);
+
+class AdaptiveScaffoldApp extends StatefulWidget {
+  final String? activeChat;
```

Comment:
idem. If it is `id`, please keep the `id`

Reply:
Fixed